src/components/Utils/orderResource.js

```
@@ -44,11 +44,18 @@ export const createOrEditOrderResource = (orderFormValues, mutator) => {
   return saveOrder(clonedOrder, mutator);
 };
 
-export const cloneOrder = (order, mutator, lines) => {
-  const clonedOrder = omit(
-    order,
-    ['id', 'adjustment', 'metadata', 'poNumber', 'poNumberPrefix', 'poNumberSuffix', 'workflowStatus', 'compositePoLines'],
-  );
+export const cloneOrder = async (order, mutator, orderNumberMutator, lines) => {
+  const orderNumberResponse = await orderNumberMutator.GET();
+  const poNumber = orderNumberResponse?.poNumber;
```

maybe throw here if `poNumber` is undefined?